scripts/functions/grafana/folders.py

```
@@ -23,16 +23,15 @@ def create_folder(folder_name, parent_folder_uid = '', recreate = False):
     folder_uid = folder_name + FOLDER_BASE_UID
     if recreate:
         delete_folder(folder_uid)
-    return __create_folder({'uid': folder_uid, 'title': folder_name, 'parentUid' : parent_folder_uid})
+    __create_folder({'uid': folder_uid, 'title': folder_name, 'parentUid' : parent_folder_uid})
 
 def __create_folder(json_data):
     result = None
     try:
         print("Create folder : %s" % json_data['title'])
         result = requests.post(GRAFANA_FOLDERS_URL, json=json_data, headers=constant.GRAFANA_HEADERS, cookies=constant.GRAFANA_COOKIES)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)